src/update_ram.c

```diff
@@ -42,20 +52,33 @@ void RAMFUNCTION wolfBoot_start(void)
 {
     int active, ret = 0;
     struct wolfBoot_image os_image;
-    uint32_t* load_address = (uint32_t*)WOLFBOOT_LOAD_ADDRESS;
+    uint32_t* load_address = (uint32_t*)kernel_addr;
     uint8_t* image_ptr;
     uint8_t p_state;
 #ifdef MMU
     uint32_t* dts_address = NULL;
 #endif
 
-    active = wolfBoot_dualboot_candidate();
+    Print(L"kernel: %x\n", kernel_addr);
```

Comment:
Please wrap in `#ifdef PLATFORM_X86_64_EFI`

Reply:
Fixed - removed debug `Print`s